docs/knowledge_base/groups.md

```diff
@@ -6,8 +6,8 @@ Phylum groups allow users to view/submit projects and analysis jobs in a shared
 2. [Create a project](../cli/commands/phylum_project_create.md) using the `--group` option and your group name
 3. [Analyze](../cli/commands/phylum_analyze.md) the desired dependency files
 
-Any user that is a member of the group will be able to access the analysis results.
+Any user that is a member of the group will be able to access the analysis results. Group administrators can add/promote members, demote administrators, and set group policy that applies to all projects in the group.
 
-Group administration for adding/removing users, etc. can be found in the `Groups Admin` view on the [Phylum UI](https://app.phylum.io/auth/login).
+Group administration for adding/removing users, etc. can be found in the `Groups` view on the [Phylum UI](https://app.phylum.io/auth/login).
 
-![user menu](../../assets/user-menu.png)
```

Comment:
This image is used in one other spot, for [the "API Keys" page](https://docs.phylum.io/knowledge_base/api-keys), so it is good that is was not deleted. It too is slightly off now with UI changes, but perhaps not so much that it matters for the context on that page.

Reply:
That is why I left it, but your guilt has prompted me to do the correct thing and update it 😝